app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -658,7 +659,7 @@ class _TextInputWidget extends ConsumerWidget {
           });
         }
       },
-      textInputAction: TextInputAction.send,
+      textInputAction: TextInputAction.newline,
```

Comment:
but doesn't this mean we always have a newline? I think on mobile, we do want to submit-with-enter ...

I'd like to have @gtalha07 's take on this please...

Reply:
Agree, could make this applied according to platform. As for desktop, the ```enter``` key should support sending.

Reply:
@gnunicorn @gtalha07 
Thanks for sharing your review. I will do below changes.

Mobile Platform:
- Message will send directly from keyboard send action as it was there previously

Desktop Platform:
- Message will add newline by shift+enter key
- Enter key will send message

Let me know if I missed or miss understood anything

Reply:
sounds good to me...

Reply:
Thanks @gtalha07.

I have did required changes and pushed the latest code. 

Can you please review and share your feedback or approval on this work.